src/misc/LoadingLine/LoadingLine.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Lottie from 'react-lottie-player/dist/LottiePlayerLight';
+import { useTranslation } from 'react-i18next';
+import { Grid, Typography } from '@material-ui/core';
+
+export const LoadingLine = (props) => {
+  const { t } = useTranslation();
```

Comment:
The component should not depend on react-i18next.
Could you please remove the `useTranslation` hook usage and use props to handle title label?